lisk-dex-electron/src/PlaceOrder.jsx

```diff
@@ -31,11 +31,18 @@ export default class PlaceOrder extends React.Component {
     });
   }
 
+  getOrderType = () => this.state.marketMode ? "market" : "limit";
+
   handleSubmit(event) {
     event.preventDefault();
     this.clearErrors();
     if (this.validateOrder()) {
-      this.placeOrder();
+      const confirmed = window.confirm(`Are you sure, you want to place this ${this.getOrderType()} order?`);
+      if (confirmed) {
+        this.clearErrors();
+        this.clearValues();
+        this.placeOrder();
```

Comment:
We should place the order because clearing the values and errors, otherwise the values like `amount` and `price` will be empty when the order is constructed.

Reply:
Ohh...😨, clearValues was after placeOrder when I tested it... maybe I mistakenly moved it before placeOrder 🤒🤒🤒🤒🤒

Reply:
Will fix it asap and push it